packages/bind.rb

```diff
@@ -3,38 +3,43 @@
 class Bind < Package
   description 'BIND is open source software that enables you to publish your Domain Name System (DNS) information on the Internet, and to resolve DNS queries for your users.'
   homepage 'https://www.isc.org/downloads/bind/'
-  version '9.17.3'
+  version '9.17.8'
   compatibility 'all'
   source_url 'file:///dev/null'
   source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.17.3-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.17.3-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.17.3-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.17.3-chromeos-x86_64.tar.xz',
+     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.17.8-chromeos-armv7l.tar.xz',
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.17.8-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.17.8-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/bind-9.17.8-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'b7dd57b241a5ef20ff4ecc36b90e442c3ae35b5f60aa890791226f54bfe188c8',
-     armv7l: 'b7dd57b241a5ef20ff4ecc36b90e442c3ae35b5f60aa890791226f54bfe188c8',
-       i686: 'a548e1980be640b0f541df3a1a4ad6f61483e39ba323b17159dccc68cd47286c',
-     x86_64: '60b6dabf560447184b9022188c74965ec22fd40becbf49d526b5ea36f0cc2c39',
+     aarch64: 'c757c886c9760396405166ee258b1953935ea3a61e40e6490c44aecbf64a5041',
+      armv7l: 'c757c886c9760396405166ee258b1953935ea3a61e40e6490c44aecbf64a5041',
+        i686: '54a28fd25b2602ac2fec3781e148c6ce9222537978b6ab79bc9457a0ab5ce534',
+      x86_64: '0d5568f9e16d5b76fd8da2815efce5dbbecb435ee99e8757141c10e8edd9539c',
   })
 
   depends_on 'libcap'
   depends_on 'libseccomp'
   depends_on 'libuv'
+  depends_on 'libidn2'
+  depends_on 'libidn2'
```

Comment:
I'm I seeing double?

Reply:
hmm. deleting that. I thought I had missed it.